CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Ewon Flexy Extensions Library Changelog
 
+## Version 1.8.0
+### Features
+- Added support for topic name in tag creation
+  - Added helper methods for persistent MEM tag creation
```

Comment:
extra space

Reply:
This is intended to be a sub-bullet, because persistent mem tag creation is reliant upon support for the topic name